### PR TITLE
Differ overtime chillout message for friday

### PR DESCRIPTION
**Pull request by ZuzannaSt**
Hi, 
I have a proposition to differ the overtime message body for notifications sent on Fridays, which now says "chill out tomorrow". As most of us actually chill outs on Saturdays, I thought that we could send emails with the "chill out next week" message, 😄 

Let me know what you think,
Cheers!

EDIT:
...also:
+ added missing translations to email templates
+ added `aggregate_failures` option to the specs with grouped expectations

### CODE REVIEW

**Comment by lubieniebieski** [comment]
what about working during the weekend? do we do something then?

File: lib/mailer.rb [hunk 1]
```diff
@@ -8,13 +8,15 @@ def self.weekly_to_office(data = {})
   end
 
   def self.daily_overtime_to_user(email, data = {})
+    week_day = data.fetch(:week_day)
```
Comment (lubieniebieski) [inline lib/mailer.rb]: shouldn't we test the scenario where `week_day` isn't part of the `data`?

File: spec/mailer_spec.rb [hunk 2]
```diff
@@ -28,5 +28,21 @@
       expect(email.body).to include('2 godzin i 30 minut')
       expect(email.body).to include('2 hours and 30 minutes')
     end
+
+    context 'when its friday' do
```
Comment (lubieniebieski) [inline spec/mailer_spec.rb]: `it's`

File: spec/mailer_spec.rb [hunk 3]
```diff
@@ -28,5 +28,21 @@
...
+      let(:week_day) { 5 }
+
+      it 'sends an email including overtime chillout message' do
+        expect(email.body).to include('Zwolnij trochę w przyszłym tygodniu :)')
+        expect(email.body).to include('Chill out next week :)')
+      end
+    end
+
+    context 'when its not friday' do
+      it 'sends an email including overtime chillout message' do
```
Comment (lubieniebieski) [inline spec/mailer_spec.rb]: ... indicating that there's a weekend ahead

**Comment by ZuzannaSt** [comment]
@lubieniebieski if working on weekends is something common for us, than what about saying something like "chill out tomorrow or next week"? Does it (or the whole idea) make sense to you? :)

File: lib/mailer.rb (hunk 1 above)
Comment (ZuzannaSt) [inline lib/mailer.rb]: would this scenario be a valid one? I mean, the `week_day` is always passed to the Mailer as it's based on the `Date.today.wday` value

**Comment by ZuzannaSt** [comment]
@lubieniebieski second thought mebe the condition shouldn't be set only for friday but for the whole friday - sunday period? 
Let's say that: 
1. Someone already has overtime on Friday - he probably does want to take a day off on Saturday and should be informed to chill out
2. Someone didn't work on Friday, but did on Saturday and Toggled some overtime - he should be informed to chill out in the next week aswell :) 
Not sure what to do with Sundays if we go this way. Maybe keep the old "chill out tomorrow" message?

**Comment by ZuzannaSt** [comment]
@lubieniebieski Also, I've noticed that we handle working on weekends separately from the overtime reports: https://github.com/netguru/ng-toggl-notifier/blob/master/lib/daily_notifier.rb#L22
https://github.com/netguru/ng-toggl-notifier/blob/master/templates/weekend_day_to_user.txt

**Comment by ZuzannaSt** [comment]
🏓  @lubieniebieski

File: lib/mailer.rb [hunk 4]
```diff
@@ -50,5 +52,16 @@ def self.render(data = {})
...
 
-  private_class_method :render
+  def self.overtime_chillout_message(week_day)
+    {
+      pl: friday?(week_day) ? 'w przyszłym tygodniu' : 'jutro',
+      en: friday?(week_day) ? 'next week' : 'tomorrow'
+    }
+  end
+
+  def self.friday?(week_day_number)
```
Comment (lubieniebieski) [inline lib/mailer.rb]: if we would pass `Data` object here, we could use methods like `date.friday?`, etc. as they are built in https://ruby-doc.org/stdlib-2.3.1/libdoc/date/rdoc/Date.html#method-i-friday-3F and they would be no necessity to define own methods.

File: templates/long_dinner.txt [hunk 5]
```diff
@@ -1,5 +1,11 @@
...
-zauważyliśmy że masz za długi wpis z obiadem, pamiętaj że maksimum to 30 minut. Proszę popraw :)
+zauważyliśmy że masz za długi wpis z obiadem. Pamiętaj że maksimum to 30 minut. Proszę popraw :)
 Tutaj więcej info:
 https://netguru.atlassian.net/wiki/pages/viewpage.action?pageId=3375489
+
+Hey,
+
+we noticed that you have a long dinner entry. Remember that the maximum is 30 minutes. Please, correct this entry :)
+Here's more info:
+https://netguru.atlassian.net/wiki/pages/viewpage.action?pageId=3375489
```
Comment (lubieniebieski) [inline templates/long_dinner.txt]: let's make it an `ENV` var maybe? it's an open repo

File: templates/long_dinner.txt [hunk 6]
```diff
@@ -1,5 +1,11 @@
 Hej,
 
-zauważyliśmy że masz za długi wpis z obiadem, pamiętaj że maksimum to 30 minut. Proszę popraw :)
+zauważyliśmy że masz za długi wpis z obiadem. Pamiętaj że maksimum to 30 minut. Proszę popraw :)
 Tutaj więcej info:
-https://netguru.atlassian.net/wiki/pages/viewpage.action?pageId=3375489
+ENV['TIME_TRACKING_FAQ']
```
Comment (lubieniebieski) [inline templates/long_dinner.txt]: please add test for these envs here, not sure if the would be parsed properly

Comment (ZuzannaSt) [inline templates/long_dinner.txt]: Fixed :)

**Comment by ZuzannaSt** [comment]
@lubieniebieski 🏓

**Pull request merged**